ams/server/src/main/java/com/netease/arctic/server/dashboard/controller/TableController.java

```diff
@@ -423,12 +424,18 @@ public void getTablePartitions(Context ctx) {
     String catalog = ctx.pathParam("catalog");
     String database = ctx.pathParam("db");
     String table = ctx.pathParam("table");
+    String keywords = ctx.queryParamAsClass("keywords", String.class).getOrDefault("");
```

Comment:
This Http API is `/tables/catalogs/{catalog}/dbs/{db}/tables/{table}/partitions?keywords=xxxxx`.

How about changing `keywords` to `filter`?